src/main/java/org/openstreetmap/atlas/checks/validation/linear/edges/MalformedRoundaboutCheck.java

```diff
@@ -163,6 +175,21 @@ private boolean intersectsWithEnclosedGeometry(final Polygon polygon, final Edge
                         || polygon.fullyGeometricallyEncloses(polyline));
     }
 
+    /**
+     * Checks if {@link AtlasObject} contains synthetic boundary Node
+     * 
+     * @param object
+     * @return true if roundabout contains synthetic boundary Node.
+     */
+    private boolean isEdgeWithSyntheticBoundaryNode(final AtlasObject object)
+    {
+        return object instanceof Edge
```

Comment:
This is not needed, unless sonar is throwing a fit. The object has already been verified to be an Edge in validCheckForObject.

Reply:
object check removed